modules/mri_violations/README.md

```diff
@@ -0,0 +1,64 @@
+# MRI violations
+
+## Purpose
+
+The MRI violations module provides a front end to view information
+about scans that were flagged as violations of the imaging protocol
+by the imaging pipeline scripts.
+
+## Intended Users
+
+The MRI violations module is primarily used by DCC staff who are
+investigating why some scans were flagged as violations of the
+imaging protocol.
+
+## Scope
+
+The MRI violations module shows a summary of the MRI violations
+(one image per row) identified by the imaging pipeline scripts.
+Depending on the type of violation, clicking on the scan's problem
+will redirect the user to pages with more information regarding
+the identified violation. The different type of violation are:
+
+- Could not identify scan type
+- Protocol Violation
+- Candidate Mismatch
+
+Additionally, the violations "Resolution Status" column can be
+updated once the violation has been resolved. Available options
+are:
+
+- Unresolved
+- Reran
+- Emailed site/pending
+- Inserted
+- Rejected
+- Inserted with flag
+- Other
```

Comment:
I think these needs descriptions of what they mean and when they should be used, as well as if they're manual or automatically (somehow?) resolved.

Reply:
Thanks! I'll ask Leigh for these as I am not sure myself...